apis/swagger.yml

```diff
@@ -227,6 +227,53 @@ paths:
       parameters:
         - $ref: "#/parameters/imageid"
 
+  /images/{imageid}/history:
+    get:
+      summary: "Get an image's history"
+      description: "Return the parent layers about image"
+      operationId: "ImageHistory"
+      produces:
+        - "application/json"
+      responses:
+        200:
+          description: "no error"
+          schema:
+            type: "array"
+            items:
+              type: "object"
```

Comment:
Could you encapsulate the whole object into an `ImageHistoryRespone`？
Maybe this would be much nicer and this will be a separate file in apis/types.